Challenge_2/Python/Papersquares/Challenge2uniquevalue.py

```diff
@@ -0,0 +1,8 @@
+
+list = [2,3,4,2,3,5,4,6,4,6,9,'a',10,9,8,7,8,10,7]
+
+newlist = []
```

Comment:
I don't know what 0(n^2) is. 

in terms of list comprehension I had to do some research but I figured out

newlist = [ ITEM for ITEM in LIST if LIST.COUNT(ITEM) ==1]